Spanner/tests/System/PgQueryTest.php

```diff
@@ -200,32 +203,56 @@ public function testBindFloat64Parameter()
         $this->assertCount(2, iterator_to_array($res));
     }
 
-    public function testBindFloat64ParameterNull()
+    public function testBindFloat32Parameter()
     {
+        $db = self::$database;
+
+        $res = $db->execute('SELECT * FROM ' . self::TABLE_NAME . ' WHERE weight = $1', [
+            'parameters' => [
+                'p1' => 1.234
```

Comment:
Can we move this value to a variable since it is used multiple places?

Reply:
Sure, but its used at one place only.